content/en/docs/reference/labels-annotations-taints/audit-annotations.md

```diff
@@ -73,3 +73,37 @@ Example: `authorization.k8s.io/decision: "Human-readable reason for the decision
 This annotation gives reason for the [decision](#authorization-k8s-io-decision) in Kubernetes audit logs.
 
 See [Auditing](/docs/tasks/debug-application-cluster/audit/) for more information.
+
+## missing-san.invalid-cert.kubernetes.io/$hostname
+
+Example: `missing-san.invalid-cert.kubernetes.io/example-svc.example-namespace.svc: "relies on a legacy Common Name field instead of the SAN extension for subject validation"`
+
+Used by Kubernetes version v1.24 and later
+
+This annotation indicates an admission or conversion webhook or aggregated API server
+is using an invalid certificate that is missing `subjectAltNames`.
+Support for these certificates was disabled by default in go1.15 / Kubernetes 1.19,
+and removed in go1.17 / Kubernetes 1.23.
```

Comment:
Are the go versions relevant to the reader here?

Reply:
I guess not, dropped